linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/Emitter.scala

```diff
@@ -252,12 +252,16 @@ final class Emitter private (config: CommonPhaseConfig,
   private def emitInitializeL0(): js.Tree = {
     implicit val pos = Position.NoPosition
 
-    // $L0 = new RuntimeLong(0, 0)
-    js.Assign(
-        jsGen.envField("L0"),
-        js.New(jsGen.encodeClassVar(LongImpl.RuntimeLongClass),
-            List(js.IntLiteral(0), js.IntLiteral(0)))
-    )
+    if (jsGen.useBigIntForLongs) {
```

Comment:
Have you considered putting this condition inside `emitInternal`? Seems cleaner to me.

Reply:
Done. I also added a first commit bumping the timeouts on the CI, because I've noticed several spurious timeouts on the master branch recently.